src/yunohost/regenconf.py

```diff
@@ -185,6 +186,20 @@ def _pre_call(name, priority, path, args):
         succeed_regen = {}
         failed_regen = {}
 
+        # Here we are doing some weird legacy shit
+        # The thing is, on some very old or specific setup, the sshd_config file
+        # was absolutely not managed by the regenconf ...
+        # But we now want to make sure that this file is managed.
+        # However, we don't want to overwrite a specific custom sshd_config
+        # which may make the admin unhappy ...
+        # So : if the hash for this file does not exists, we set the hash as the
+        # hash of the pending configuration ...
```

Comment:
```suggestion
        # hash of the current configuration ...
```

Reply:
Because if we put the current hash, the conf is going to be replaced by the new configuration ... but that's not what we want, we want the conf to be flagged as manually modified

Reply:
I didn't understand the point of this hash, my bad